package.json

```diff
@@ -28,14 +28,61 @@
     "@angular/core": ">=4.3.6",
     "@angular/forms": ">=4.3.6",
     "@angular/router": ">=4.3.6",
-    "@skyux/core": "^3.5.1",
-    "@skyux/i18n": "^3.5.0",
-    "@skyux/indicators": "^3.0.0"
+    "@skyux-sdk/builder": "^3.7.0",
```

Comment:
Looks like the migration added some peers.

Reply:
I fixed this :)